src/index.js

```diff
@@ -1,2 +1,2 @@
-export Notification from './notification';
-export NotificationStack from './notificationStack';
+export { Notification } from './notification';
+export { NotificationStack } from './notificationStack';
```

Comment:
`export { Notification as default } from './notification';` <- default is flipped onto the other side.